### ISSUE TITLE
[Tokenizer]: [A11y]  Tokens Not Read in JAWS VPC Mode

### ISSUE THREAD

**Issue by sleric1024** [SAP/ui5-webcomponents#11327]
### Describe the bug

The tokens needs to be with the role list and not list box, this will enable it to read in VPC mode.

### Isolated Example

https://stackblitz.com/edit/github-dr9gxkcb?file=src%2FApp.tsx

### Reproduction steps

1. Navigate to the demo link
2. Using JAWS VPC Mode
3. Tokens Not Read in JAWS VPC Mode

### Expected Behaviour

In VPC mode, tokens must be read. Assigning a role with list would facilitate this process.

### Screenshots or Videos

![Image](https://github.com/user-attachments/assets/5a6cc215-22ce-4830-ab0d-3588c5aa4c17)

### UI5 Web Components for React Version

2.9.0

### UI5 Web Components Version

2.9.-

### Browser

Chrome

### Operating System

Windows

### Additional Context

This issue is created by A11y Expert from Successfactors for b2505 feature release. Could you please kindly help to check and fix this, thanks

### Relevant log output

```shell

```

### Organization

SF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-rl , 

The Tokens in the Tokenizer are not announced in VPC mode - I just tested in JAWS 2024 and NVDA. 

Kind regards, 

Georgi
SAP Development Support

**Comment by elenastoyanovaa** [comment]
Hello,

The screenshot you provided does not correspond to the latest screen reader specification. 
Here is a screenshot of the latest screen reader spec:

<img width="643" alt="Image" src="https://github.com/user-attachments/assets/118513e1-d842-498d-b3a7-96964cdeb62e" />

What is more there is an ongoing spec update FIORITECHP1-33413 and is not yet evaluated. You can track the final specification and screen reader implementation there.

Kind Regards,
Elena

**Comment by judith9209** [comment]
Hi, would you recheck the description and recording of jira ticket WEF-162686 ? When you are in VPC mode, the tokens are not accessible. Is this handled with  FIORITECHP1-33413? Many thanks!

**Comment by hristop** [comment]
Hi @judith9209 ,

I have added comment in 33413 tagging the responsible design colleagues. WEF-162686 is linked to the BLI so when they align the with the ACC team, they should have it in mind. 
I am closing this incident. Please continue monitoring the progress in FIORITECHP1-33413 and the corresponding linked BLIs.

Best Regards,
Hristo
